docker-x64/Dockerfile

```
@@ -41,6 +47,16 @@ COPY requirements /requirements
 
 RUN pip install -r requirements.txt
 
+# Install Go
+RUN curl -sL -o /tmp/golang.tar.gz https://go.dev/dl/go$GO_VERSION.linux-amd64.tar.gz \
+  && echo "$GO_SHA256_LINUX_AMD64  /tmp/golang.tar.gz" | sha256sum --check \
+  && tar -C /usr/local -xzf /tmp/golang.tar.gz \
+  && rm -f /tmp/golang.tar.gz
+ENV GOPATH /go
+ENV GO_VERSION $GO_VERSION
+ENV PATH="/usr/local/go/bin:${PATH}"
+ENV PATH="${GOPATH}/bin:${PATH}"
```

Can be done in one statement to save one layers.

```suggestion
ENV PATH="/go/bin:/usr/local/go/bin:${PATH}"
```

---

Good point, thanks !